### DISCUSSION TITLE
Need Help - Request failed With status Code 500

### DISCUSSION THREAD

**Discussion by cassanovade** [Security-Onion-Solutions/securityonion#8873] [Unsupported Versions]
Hi Everyone..., 

Im new with SO, i've install SO V.2.3.140 with Eval Mode a month ago. At first, it running okay. But about week ago, i found Error code 500 when i open alert, dashboard, hunt and some other module. Also eror 404 page not found when i open Kibana.

i've try some suggest in help Documentation.

1. at first, i check the log, but i can't access into root log folder (some of log folder is create for root only), is it in Eval Mode we can't know our root user? because when i remembered at first install, there is no step shown our root user or password typing. There is only step where i create user to access the SO. When i access root folder and other (with root access), always permission denied.
im also check docker log with "sudo docker logs so-kibana" and there is the last result :

=============

`[2022-10-07T03:54:00.685+00:00][ERROR][savedobjects-service] [.kibana_task_manager] Unexpected Elasticsearch ResponseError: statusCode: 429, method: GET, url: /.kibana_task_manager%2C.kibana_task_manager_8.4.1?ignore_unavailable=true error: [circuit_breaking_exception]: [parent] Data too large, data for [<http_request>] would be [608589784/580.3mb], which is larger than the limit of [597688320/570mb], real usage: [608589784/580.3mb], new bytes reserved: [0/0b], usages [inflight_requests=0/0b, request=0/0b, fielddata=0/0b, eql_sequence=0/0b, model_inference=0/0b],
 FATAL  Error: Unable to complete saved object migrations for the [.kibana] index. Please check the health of your Elasticsearch cluster and try again. Unexpected Elasticsearch ResponseError: statusCode: 429, method: GET, url: /.kibana%2C.kibana_8.4.1?ignore_unavailable=true error: [circuit_breaking_exception]: [parent] Data too large, data for [<http_request>] would be [604443512/576.4mb], which is larger than the limit of [597688320/570mb], real usage: [604443512/576.4mb], new bytes reserved: [0/0b], usages [inflight_requests=0/0b, request=0/0b, fielddata=0/0b, eql_sequence=0/0b, model_inference=0/0b`

============

2. next try, i've running sudo-soup to update SO. now im running with SO V2.3.170, but the error still same. "Request failed With status Code" also Kibana "404 page not found"
3. next try about kibana eror 404, i've try to so-elastic-auth-password-reset so_kibana. In that progress found some failed process.
![fail1](https://user-images.githubusercontent.com/62920693/194469099-9af336fd-a5e2-4b42-aad0-de8f19c56f37.png)

![fail 2](https://user-images.githubusercontent.com/62920693/194469120-955207ea-a436-49fe-876b-12fb54adee9d.png)

and then also test with "so-elastic-auth true" then restart Kibana, still the same error 404 page not found.
 
any other suggestion about my case?

**Comment by dougburks** [comment]
> at first, i check the log, but i can't access into root log folder (some of log folder is create for root only), is it in Eval Mode we can't know our root user? 

You should be able to escalate to `root` using `sudo -i`.

> [2022-10-07T03:54:00.685+00:00][ERROR][savedobjects-service] [.kibana_task_manager] Unexpected Elasticsearch ResponseError: statusCode: 429, method: GET, url: /.kibana_task_manager%2C.kibana_task_manager_8.4.1?ignore_unavailable=true error: [circuit_breaking_exception]: [parent] Data too large, data for [<http_request>] would be [608589784/580.3mb], which is larger than the limit of [597688320/570mb], real usage: [608589784/580.3mb], new bytes reserved: [0/0b], usages [inflight_requests=0/0b, request=0/0b, fielddata=0/0b, eql_sequence=0/0b, model_inference=0/0b], FATAL Error: Unable to complete saved object migrations for the [.kibana] index. Please check the health of your Elasticsearch cluster and try again. Unexpected Elasticsearch ResponseError: statusCode: 429, method: GET, url: /.kibana%2C.kibana_8.4.1?ignore_unavailable=true error: [circuit_breaking_exception]: [parent] Data too large, data for [<http_request>] would be [604443512/576.4mb], which is larger than the limit of [597688320/570mb], real usage: [604443512/576.4mb], new bytes reserved: [0/0b], usages [inflight_requests=0/0b, request=0/0b, fielddata=0/0b, eql_sequence=0/0b, model_inference=0/0b

Have you modified Elasticsearch heap size?
https://docs.securityonion.net/en/2.3/elasticsearch.html#heap-size

> Im new with SO, i've install SO V.2.3.140 with Eval Mode a month ago.

Since this is a relatively new Eval installation, then if all else fails you might just want to perform a fresh installation of the latest version (currently 2.3.170):
https://blog.securityonion.net/2022/09/security-onion-23170-now-available.html

**Reply by cassanovade** [reply]
> You should be able to escalate to root using sudo -i.

owh, thanks, i can escalate to root user now.

> Have you modified Elasticsearch heap size?
https://docs.securityonion.net/en/2.3/elasticsearch.html#heap-size

hmmm, let me try that. 

> Since this is a relatively new Eval installation, then if all else fails you might just want to perform a fresh installation of the latest version (currently 2.3.170):
https://blog.securityonion.net/2022/09/security-onion-23170-now-available.html

> 2. next try, i've running sudo-soup to update SO. now im running with SO V2.3.170, but the error still same. "Request failed With status Code" also Kibana "404 page not found"

actually, at my statement number 2. im already update the SO into 2.3.170, but the problem still same.

**Reply by weslambert** [reply]
Are you still experiencing an issue?
